tool/tctl/common/edit_command.go

```diff
@@ -69,6 +73,26 @@ func (e *EditCommand) TryRun(ctx context.Context, cmd string, client auth.Client
 	return true, trace.Wrap(err)
 }
 
+func (e *EditCommand) runEditor(ctx context.Context, name string) error {
+	if e.editor != nil {
+		return trace.Wrap(e.editor(name))
+	}
+
+	args := strings.Fields(editor())
```

Comment:
Maybe change either `editor()` or `e.editor` name a bit, so there's less chance for consufion? Fx `editor()` -> `getEditorName()`?